web/src/components/dashboards/perses/dashboard-api.ts

```diff
@@ -0,0 +1,51 @@
+import { DashboardResource } from '@perses-dev/core';
+import buildURL from './perses/url-builder';
+import { consoleFetch } from '@openshift-console/dynamic-plugin-sdk';
+import { useMutation, UseMutationResult, useQueryClient } from '@tanstack/react-query';
+
+const resource = 'dashboards';
+
+const updateDashboard = async (entity: DashboardResource): Promise<DashboardResource> => {
+  const HTTPMethodPUT = 'PUT';
+  const HTTPHeader: Record<string, string> = {
+    'Content-Type': 'application/json',
+    Accept: 'application/json',
+  };
+  const url = buildURL({
+    resource: resource,
+    project: entity.metadata.project,
+    name: entity.metadata.name,
+  });
+
+  const response = await consoleFetch(url, {
```

Comment:
The console fetch has a put method, so we don't have to set the method manually

Reply:
Updated to use `consoleFetchJSON.put(...)`.